README.md

```diff
@@ -216,6 +216,7 @@ You may need super-user privileges.
 SingleM also has the following non-Python dependencies:
 * [smafa](https://github.com/wwood/smafa) >= 0.5.0
 * [VSEARCH](https://github.com/torognes/vsearch)
+* [sed](https://github.com/mirror/sed)
```

Comment:
https://www.gnu.org/software/sed/ is a better link. Also worth stating that sed comes standard on most linux systems, so that people don't have to worry.

Let me know when you want a re-review.

Reply:
I think it should be all good now. What do you think?